tools/pd-simulator/main.go

```diff
@@ -43,7 +43,7 @@ import (
 )
 
 var (
-	pdAddr                      = flag.String("pd", "", "pd address")
+	pdAddr                      = flag.String("pd-endpoints", "", "pd address")
 	configFile                  = flag.String("config", "conf/simconfig.toml", "config file")
```

Comment:
```suggestion
	configFile                  = flag.String("config", "../conf/simconfig.toml", "config file")
```
BTW, In general, developers always run `simulator` in the `bin` directory, so I thought it's better to change it.

Reply:
it will fix it by next pr